[2024-06-03T10:49:06.771+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T10:49:06.795+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: city_data_workflow_DAG_final.delete_data_in_postgres manual__2024-06-03T10:48:40.858423+00:00 [queued]>
[2024-06-03T10:49:06.805+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: city_data_workflow_DAG_final.delete_data_in_postgres manual__2024-06-03T10:48:40.858423+00:00 [queued]>
[2024-06-03T10:49:06.806+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-03T10:49:06.819+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): delete_data_in_postgres> on 2024-06-03 10:48:40.858423+00:00
[2024-06-03T10:49:06.827+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'city_data_workflow_DAG_final', 'delete_data_in_postgres', 'manual__2024-06-03T10:48:40.858423+00:00', '--job-id', '1026', '--raw', '--subdir', 'DAGS_FOLDER/tasks_flow.py', '--cfg-path', '/tmp/tmprbytnw9v']
[2024-06-03T10:49:06.830+0000] {standard_task_runner.py:91} INFO - Job 1026: Subtask delete_data_in_postgres
[2024-06-03T10:49:06.833+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=433) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-03T10:49:06.834+0000] {standard_task_runner.py:63} INFO - Started process 435 to run task
[2024-06-03T10:49:06.889+0000] {task_command.py:426} INFO - Running <TaskInstance: city_data_workflow_DAG_final.delete_data_in_postgres manual__2024-06-03T10:48:40.858423+00:00 [running]> on host cd9caf1c47f0
[2024-06-03T10:49:06.986+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Khang' AIRFLOW_CTX_DAG_ID='city_data_workflow_DAG_final' AIRFLOW_CTX_TASK_ID='delete_data_in_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-06-03T10:48:40.858423+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-03T10:48:40.858423+00:00'
[2024-06-03T10:49:06.988+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T10:49:06.999+0000] {sql.py:276} INFO - Executing: DELETE FROM cities;
[2024-06-03T10:49:07.012+0000] {base.py:84} INFO - Using connection ID 'postgres_connection_id' for task execution.
[2024-06-03T10:49:07.023+0000] {base.py:84} INFO - Using connection ID 'postgres_connection_id' for task execution.
[2024-06-03T10:49:07.032+0000] {sql.py:470} INFO - Running statement: DELETE FROM cities;, parameters: None
[2024-06-03T10:49:07.033+0000] {sql.py:479} INFO - Rows affected: 1
[2024-06-03T10:49:07.036+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T10:49:07.050+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=city_data_workflow_DAG_final, task_id=delete_data_in_postgres, run_id=manual__2024-06-03T10:48:40.858423+00:00, execution_date=20240603T104840, start_date=20240603T104906, end_date=20240603T104907
[2024-06-03T10:49:07.089+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T10:49:07.138+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T10:49:07.142+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
